apps/blog/src/app/single-post/single-post.component.ts

```diff
@@ -0,0 +1,23 @@
+import { ChangeDetectionStrategy, Component, OnInit, Input } from '@angular/core';
+import { Observable } from 'rxjs';
+import { Post } from '../form/form.component';
+import { FormService } from '../form.service';
+import { ActivatedRoute } from '@angular/router';
+
+
+
+@Component({
+  selector: 'codelab-single-post',
+  templateUrl: './single-post.component.html',
+  styleUrls: ['./single-post.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class SinglePostComponent implements OnInit {
+
+ @Input() post: Post;
+ @Input() full: boolean;
+ @Input() ngClass: any;
```

Comment:
ngClass -> className? It's a string. 
Why fo you need this?

Reply:
To change className for different use cases of this component?
Is there a better way?